website/docs/BoilerplateProjects.md

```diff
@@ -52,6 +52,20 @@ Boilerplate project to run WebdriverIO tests with Jasmine using Babel features a
   - Page Object Pattern
   - Sauce Labs integration
 
+## [17thSep/WebdriverIO_Master](https://github.com/17thSep/WebdriverIO_Master)
```

Comment:
could we move and join it to https://github.com/webdriverio/webdriverio/pull/6425/files#diff-5047c2de25222857de23ee59f93e410374eb238d9439bd477b0b2e6c34d4ef89R270 ?

Reply:
I am not quite following, can you please explain in detail

Reply:
Move this to https://github.com/17thSep/webdriverio/blob/patch-2/website/docs/BoilerplateProjects.md#17thsepmock-and-spies

Reply:
Done

Reply:
looks good?